.eslintrc

```diff
@@ -1,187 +1,9 @@
-    "no-func-assign": [2],
-    "no-implied-eval": [2],
-    "no-inline-comments": [0],
-    "no-inner-declarations": [2, "functions"],
-    "no-invalid-regexp": [2],
-    "no-irregular-whitespace": [2],
-    "no-iterator": [2],
-    "no-label-var": [2],
-    "no-labels": [2],
-    "no-lone-blocks": [2],
-    "no-lonely-if": [2],
-    "no-loop-func": [2],
-    "no-mixed-requires": [0, false],
-    "no-mixed-spaces-and-tabs": [2, false],
-    "no-multi-spaces": [2],
-    "no-multi-str": [2],
-    "no-multiple-empty-lines": [2, {"max": 2}],
-    "no-native-reassign": [1],
-    "no-negated-in-lhs": [2],
-    "no-nested-ternary": [0],
-    "no-new": [2],
-    "no-new-func": [2],
-    "no-new-object": [2],
-    "no-new-require": [0],
-    "no-new-wrappers": [2],
-    "no-obj-calls": [2],
-    "no-octal": [2],
-    "no-octal-escape": [2],
-    "no-param-reassign": [2],
-    "no-path-concat": [0],
-    "no-plusplus": [0],
-    "no-process-env": [0],
-    "no-process-exit": [2],
-    "no-proto": [2],
-    "no-redeclare": [2],
-    "no-regex-spaces": [2],
-    "no-reserved-keys": [0],
-    "no-restricted-modules": [0],
-    "no-return-assign": [2],
-    "no-script-url": [2],
-    "no-self-compare": [0],
-    "no-sequences": [2],
-    "no-shadow": [2],
-    "no-shadow-restricted-names": [2],
-    "no-spaced-func": [2],
-    "no-sparse-arrays": [2],
-    "no-sync": [0],
-    "no-ternary": [0],
-    "no-throw-literal": [2],
-    "no-trailing-spaces": [2],
-    "no-undef-init": [2],
-    "no-undefined": [0],
-    "no-underscore-dangle": [2],
-    "no-unreachable": [2],
-    "no-unused-expressions": [2],
-    "no-unused-vars": [1, {"vars": "all", "args": "after-used"}],
-    "no-use-before-define": [2],
-    "no-void": [0],
-    "no-warning-comments": [0, {"terms": ["todo", "fixme", "xxx"], "location": "start"}],
-    "no-with": [2],
-    "one-var": [0],
-    "operator-assignment": [0, "always"],
-    "operator-linebreak": [2, "after"],
-    "padded-blocks": [0],
-    "quote-props": [0],
-    "radix": [0],
-    "semi": [2],
-    "semi-spacing": [2, {"before": false, "after": true}],
-    "sort-vars": [0],
-    "keyword-spacing": [2],
-    "space-before-function-paren": [2, {"anonymous": "always", "named": "never"}],
-    "space-before-blocks": [0, "always"],
-    "space-in-brackets": [
-      0, "never", {
-        "singleValue": true,
-        "arraysInArrays": false,
-        "arraysInObjects": false,
-        "objectsInArrays": true,
-        "objectsInObjects": true,
-        "propertyName": false
-      }
-    ],
-    "space-in-parens": [0],
-    "space-infix-ops": [2],
-    "space-unary-ops": [2, {"words": true, "nonwords": false}],
-    "spaced-line-comment": [0, "always"],
-    "strict": [2, "never"],
-    "use-isnan": [2],
-    "valid-jsdoc": [0],
-    "valid-typeof": [2],
-    "vars-on-top": [0],
-    "wrap-iife": [2],
-    "wrap-regex": [2],
-    "yoda": [2, "never", {"exceptRange": true}]
+    "no-param-reassign": 0,
```

Comment:
@boneskull you are correct. I have these changes in my git for quite a while. It was just quicker way for me.

@FredyC it is because of `mergeComposable` function. Namely these two lines:
```js
if (!isDescriptor(dstDescriptor[propName])) dstDescriptor[propName] = {};
```
and
```js
dstDescriptor.initializers = uniq(dstInitializers.filter(isFunction)
```